bokeh/plotting/figure.py

```diff
@@ -559,6 +559,16 @@ def scatter(self, *args, **kwargs):
 
         return getattr(self, markertype)(*args, **kwargs)
 
+class WebGLFigure(Figure):
+    ''' A subclass of :class:`~bokeh.models.plots.Plot` that simplifies plot
+    creation with default axes, grids, tools, etc., and support rendering
+    with WebGL, if available. This requires addition of `bokeh-gl.min.js`
+    bundle.
+
+    '''
+
+    __subtype__ = "WebGLFigure"
+    __view_model__ = "WebGLPlot"
```

Comment:
Should this be `bokeh.models.plots.WebGLPlot` instead of `WebGLFigure`? Thinking of anyone wanting to create a WebGL using `bokeh.models`